lls/main.c

```diff
@@ -500,12 +531,12 @@ lls_stage2(void *arg)
 	 * Gatekeeper or Grantor.
 	 */
 
-	/*
-	 * TODO Have a different block set up RSS on the back interface,
-	 * and classify + distribute ND packets to the LLS block using
-	 * the DPDK libraries.
-	 */
-	if (lls_conf->nd_cache.iface_enabled(net_conf, &net_conf->back)) {
+	ret = register_nd_acl_rules(&net_conf->front);
+	if (ret < 0)
+		return ret;
+
+	/* TODO Have a different block set up RSS on the back interface. */
```

Comment:
`finalize_stage2()` could test if the back interface is enabled and no block set RSS for the back interface. For this case, `finalize_stage2()` could set RSS for LLS to take care of it.